src/types/i18n.d.ts

```diff
@@ -0,0 +1,7 @@
+import { resources } from '../i18n';
+
+declare module 'react-i18next' {
```

Comment:
Isn't this redundant? https://www.npmjs.com/package/@types/react-i18next exists, and according to the link the package should have its own types